util/pingpong.c

```diff
@@ -323,6 +323,9 @@ void pp_banner_options(struct ct_pingpong *ct)
 	PP_DEBUG("  - %-20s: %s\n", "iterations", iter_msg);
 	PP_DEBUG("  - %-20s: %s\n", "provider",
 		 ct->hints->fabric_attr->prov_name);
```

Comment:
> Should we use any default provider? I can add it to my PR if you want.

No, the default behavior of `fi_pingpong` is to select the first provider returned if no hint was given. This function is here solely for debugging purposes and used to print the options that were given. If there was no provider hint given, then no use in printing it out.